docs/build/circuit-library.ipynb

```diff
@@ -0,0 +1,454 @@
+      "text/plain": [
+       "<Figure size 538.33x284.278 with 1 Axes>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.library import PauliEvolutionGate\n",
+    "from qiskit.circuit import QuantumCircuit\n",
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "\n",
+    "# Prepare an initial state with a Hamadard on the middle qubit\n",
+    "state = QuantumCircuit(3)\n",
+    "state.h(1)\n",
+    "\n",
+    "hamiltonian = SparsePauliOp([\"ZZI\", \"IZZ\"])\n",
+    "evolution = PauliEvolutionGate(hamiltonian, time=1)\n",
+    "\n",
+    "# Evolve state by appending the evolution gate\n",
+    "state.compose(evolution, inplace=True)\n",
+    "\n",
+    "state.draw('mpl')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2dad12f-535a-4f42-8ac4-dbfcfb5533cc",
+   "metadata": {},
+   "source": [
+    "Read the [`PauliEvolutionGate` API documentation](/api/qiskit/qiskit.circuit.library.PauliEvolutionGate)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0cf7122d-c3fe-41a6-936c-b3770b33f0f1",
+   "metadata": {},
+   "source": [
+    "## Benchmarking and complexity-theory circuits\n",
+    "\n",
+    "Benchmarking circuits give us a sense of how well our hardware is actually working, and complexity-theory circuits help us understand how difficult the problems we want to solve are.\n",
+    "\n",
+    "For example, the \"quantum volume\" benchmark measures how accurately a quantum computer executes a type of random quantum circuit. The score of the quantum computer increases with the size of the circuit it can reliably run. This takes into account all aspects of the computer, including qubit count, instruction fidelity, qubit connectivity, and the software stack transpiling and post-processing results. Read more about quantum volume in the original [quantum volume paper](https://arxiv.org/abs/1811.12926).\n",
```

Comment:
Nothing to change now, but QV is changing names probably this week FYI. Since we can't change the name of the original paper, may need to do some wordsmithing here once the name changes.

Reply:
Interesting, what's it changing to?